### PR TITLE
Adding GitHub Actions VS - without exceptions.

### PR DESCRIPTION
**Pull request by lemire**

### CODE REVIEW

File: .github/workflows/vs16-noexcept-ci.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: VS16-NoExcept-CI
```
Comment (jkeiser) [inline .github/workflows/vs16-noexcept-ci.yml:1]: Not something to solve here, but there *must* be a way for github ci actions to share code ...

**Comment by jkeiser** [comment]
Ah, this is because Appveyor is so overloaded as it is and we want to check exceptions off with an earlier version of Visual Studio?

**Review by jkeiser — APPROVED** [review]
Seems like a good thing to have. Should we review our use of appveyor and convert more to github actions? Appveyor is generally our long pole in CI ...

File: .github/workflows/vs16-noexcept-ci.yml (hunk 1 above)
Comment (lemire) [reply .github/workflows/vs16-noexcept-ci.yml:1]: Yes. But I find it difficult to read and debug complex yaml files. Having distinct tests be in distinct yaml files is easier to me.

**Comment by lemire** [comment]
> Ah, this is because Appveyor is so overloaded as it is and we want to check exceptions off with an earlier version of Visual Studio?

We do test with exceptions disabled on Appveyor, but only in Win32 mode as far as I can tell.

**Pull request merged**